lambdas/services/lloyd_george_stitch_job_service.py

```diff
@@ -0,0 +1,207 @@
+import os
+from datetime import datetime, timedelta
+
+from botocore.exceptions import ClientError
+from enums.dynamo_filter import AttributeOperator
+from enums.lambda_error import LambdaError
+from enums.trace_status import TraceStatus
+from models.stitch_trace import DocumentStitchJob, StitchTrace
+from pydantic import ValidationError
+from services.base.dynamo_service import DynamoDBService
+from services.base.s3_service import S3Service
+from services.document_service import DocumentService
+from utils.audit_logging_setup import LoggingService
+from utils.dynamo_query_filter_builder import DynamoQueryFilterBuilder
+from utils.exceptions import FileUploadInProgress, NoAvailableDocument
+from utils.lambda_exceptions import LGStitchServiceException
+from utils.lloyd_george_validator import LGInvalidFilesException
+
+logger = LoggingService(__name__)
+
+
+class LloydGeorgeStitchJobService:
+    def __init__(self):
+        get_document_presign_url_aws_role_arn = os.getenv("PRESIGNED_ASSUME_ROLE")
+        self.s3_service = S3Service(
+            custom_aws_role=get_document_presign_url_aws_role_arn
+        )
+        self.dynamo_service = DynamoDBService()
+        self.document_service = DocumentService()
+        self.stitch_trace_table = os.environ.get("STITCH_METADATA_DYNAMODB_NAME")
+        self.lloyd_george_table_name = os.environ.get("LLOYD_GEORGE_DYNAMODB_NAME")
+
+        self.cloudfront_url = os.environ.get("CLOUDFRONT_URL")
+        self.lloyd_george_bucket_name = os.environ.get("LLOYD_GEORGE_BUCKET_NAME")
+
+    def create_stitch_job(self, nhs_number: str) -> TraceStatus:
```

Comment:
"get_latest_stitch_trace_status" ?

Reply:
But it does create a new one if there isn't one. I'll change it to get or create.